New Entry: Cloud AWS Management Console (#1162)
Codecademy/docs: content/cloud-computing/concepts/aws-management-console/aws-management-console.md

---
Title: 'AWS Management Console'
Description: 'The AWS Management Console is a web application that is home to all the various other consoles used to manage AWS services.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Cloud Computing'
CatalogContent:
  - 'learn/foundations-of-cloud-computing'
  - 'paths/back-end-engineer-career-path'
---

The **AWS Management Console** is a web application that is home to all the various other consoles used to manage AWS services. The console home page is what's first seen when a user logs into AWS.

> The home page provides access to each service console and provides information needed to perform AWS-related tasks.

Author: @Dusch4593, I thought that was what the following bulleted list was doing.

> The home page provides access to each service console and provides information needed to perform AWS-related tasks.

Reviewer: @SSwiniarski Oh I see what you mean, now. When I read the list again, it read to me as a list of both tasks and features.

> The home page provides access to each service console and provides information needed to perform AWS-related tasks.

Author: @Dusch4593 Is there an edit you'd suggest, or does it look good as-is?

> The home page provides access to each service console and provides information needed to perform AWS-related tasks.

Reviewer 2: Is it worth mentioning that the console is user customizable in this opening section? Since there are references to this in the bullet list it might be a good high level feature to mention in the main paragraph.

The AWS Management Console provides the following:

> - Unified settings for all service consoles for things like region and screen display.

Reviewer 2: ```suggestion
- Unified settings for all service consoles, such as region and screen display.
```
